pymatgen/io/qchem/outputs.py

```diff
@@ -38,9 +38,8 @@
     have_babel = False
 
 
-__author__ = "Samuel Blau, Brandon Wood, Shyam Dwaraknath, Evan Spotte-Smith"
-__copyright__ = "Copyright 2018, The Materials Project"
-__version__ = "0.1"
+__author__ = "Brandon Wood, Samuel Blau, Shyam Dwaraknath, Evan Spotte-Smith, Ryan Kingsbury"
```

Comment:
Not sure why Brandon Wood became first here? Very minor, but originally he wrote inputs and I wrote outputs. Order should be changed back, with Ryad added on the end.

Reply:
I didn't realize the order was meaningful;  had just copy/pasted from inputs to make the list consistent across files. Reverted now.